src/States/MapViewStateDraw.cpp

```diff
@@ -154,63 +154,39 @@ void MapViewState::drawResourceInfo()
 	const auto glowIntensity = calcGlowIntensity();
 	const auto glowColor = NAS2D::Color{255, glowIntensity, glowIntensity};
 
-	// Common Metals
-	const auto commonMetalsImageRect = NAS2D::Rectangle<int>{64, 16, constants::RESOURCE_ICON_SIZE, constants::RESOURCE_ICON_SIZE};
-	renderer.drawSubImage(mUiIcons, position, commonMetalsImageRect);
-	bool shouldCommonMetalsGlow = mPlayerResources.commonMetals() <= 10;
-	auto color = shouldCommonMetalsGlow ? glowColor : NAS2D::Color::White;
-	renderer.drawText(*MAIN_FONT, std::to_string(mPlayerResources.commonMetals()), position + textOffset, color);
-
-	// Rare Metals
-	position.x() += offsetX;
-	const auto rareMetalsImageRect = NAS2D::Rectangle<int>{80, 16, constants::RESOURCE_ICON_SIZE, constants::RESOURCE_ICON_SIZE};
-	renderer.drawSubImage(mUiIcons, position, rareMetalsImageRect);
-	bool shouldRareMetalsGlow = mPlayerResources.rareMetals() <= 10;
-	color = shouldRareMetalsGlow ? glowColor : NAS2D::Color::White;
-	renderer.drawText(*MAIN_FONT, std::to_string(mPlayerResources.rareMetals()), position + textOffset, color);
-
-	// Common Minerals
-	position.x() += x + offsetX;
-	const auto commonMineralsImageRect = NAS2D::Rectangle<int>{96, 16, constants::RESOURCE_ICON_SIZE, constants::RESOURCE_ICON_SIZE};
-	renderer.drawSubImage(mUiIcons, position, commonMineralsImageRect);
-	bool shouldCommonMineralsGlow = mPlayerResources.commonMinerals() <= 10;
-	color = shouldCommonMineralsGlow ? glowColor : NAS2D::Color::White;
-	renderer.drawText(*MAIN_FONT, std::to_string(mPlayerResources.commonMinerals()), position + textOffset, color);
+	constexpr auto iconSize = constants::RESOURCE_ICON_SIZE;
+	const std::array resources{
+		std::tuple{NAS2D::Rectangle{64, 16, iconSize, iconSize}, mPlayerResources.commonMetals(), offsetX},
+		std::tuple{NAS2D::Rectangle{80, 16, iconSize, iconSize}, mPlayerResources.rareMetals(), x + offsetX},
+		std::tuple{NAS2D::Rectangle{96, 16, iconSize, iconSize}, mPlayerResources.commonMinerals(), x + offsetX},
+		std::tuple{NAS2D::Rectangle{112, 16, iconSize, iconSize}, mPlayerResources.rareMinerals(), 0},
+	};
 
-	// Rare Minerals
-	position.x() += x + offsetX;
-	const auto rareMineralsImageRect = NAS2D::Rectangle<int>{112, 16, constants::RESOURCE_ICON_SIZE, constants::RESOURCE_ICON_SIZE};
-	renderer.drawSubImage(mUiIcons, position, rareMineralsImageRect);
-	bool shouldRareMineralsGlow = mPlayerResources.rareMinerals() <= 10;
-	color = shouldRareMineralsGlow ? glowColor : NAS2D::Color::White;
-	renderer.drawText(*MAIN_FONT, std::to_string(mPlayerResources.rareMinerals()), position + textOffset, color);
+	for (const auto& [imageRect, amount, spacing] : resources) {
```

Comment:
For consistency with rest of project, there should be a newline between the for loop and the opening brace.
```cpp
for ...
{
```

Reply:
Hmm, this syntax has slipped through for a few other recent commits.